.rubocop.yml

```diff
@@ -10,7 +10,7 @@ AllCops:
 Layout/LineLength:
```

Comment:
- [ ] In Readme file, kindly remove "Microverse README Template" logo. This would help you with making your readme more professional.